queries/organization-optional.rq

```diff
@@ -8,7 +8,7 @@ SELECT
   (SAMPLE(?contactPage) AS ?contactPage)
   (SAMPLE(?regulatoryText ) AS ?regulatoryText)
   (SAMPLE(?regulatoryTextTitle ) AS ?regulatoryTextTitle)
-  (SAMPLE(?personLabel) AS ?leadBy)
+  (SAMPLE(COALESCE(?personLabel, "Not Available")) AS ?leadBy)
```

Comment:
This will break our check on the org.html and lead to inconsistent behavior for when data is missing. Over there, we check if there is a value, and if not we display `[missing data]`. The same goes for all the other values in the info box. So please undo this part.